MyDataHelpsKit/Model/Project.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  Project.swift
+//  MyDataHelpsKit
+//
+//  Created by CareEvolution on 8/11/22.
+//
+
+import Foundation
+
+/// Information about the project and its settings.
+public struct ProjectInfo: Decodable {
+    /// Unique project identifier.
+    public let id: String
+    /// Project's display name, localized based on participant's language settings.
+    public let name: String
+    /// Project's display description, localized based on participant's language settings.
+    public var description: String?
+    /// Code used by participants to enroll in this project, if code enrollment is allowed.
+    public let code: String
+    /// Project type, displayed to participants and used to filter search results.
+    public let type: ProjectType
+    /// Information about the organization for this project.
+    public let organization: Organization
+    /// Contact email for project support displayed to participants.
+    public var supportEmail: String?
+    /// Contact number for project support displayed to participants.
+    public var supportPhone: String?
+    /// A URL where participants can learn more about the project.
+    public var learnMoreURL: URL? {
+        learnMoreLink.flatMap { URL(string: $0) }
+    }
+    /// Site name or title to use as a label for the ``learnMoreURL``, localized based on participant's language settings.
+    public let learnMoreTitle: String?
+    
+    /// The raw decodable value, which may be an empty string or invalid URL.
+    private var learnMoreLink: String?
+}
+
+/// Describes the type of a MyDataHelps project.
+/// May be one of: Research Study, Wellness Program, or Clinical Program.
+public struct ProjectType: RawRepresentable, Equatable, Decodable {
```

Comment:
Any reason why this would be a struct instead of an enum?

Reply:
For forward compatibility. If this were `struct ProjectType: String`, and we introduce a new ProjectType value on the server, then the SDK would immediately fail to decode the ProjectInfo for such projects, throwing a decoding error. Using a RawRepresentable struct instead, these new ProjectType values successfully decode, simply as instances with different rawValues.

This pattern is used with most of the enum-like decodable values in the SDK. In practice, the syntax for using these structs (choosing specific values or checking for equality to specific values) looks identical to plain enums, as seen throughout the example app. e.g. you can just check for `projectInfo.type == .researchStudy`, use the shorthand dot-syntax in switch cases, etc.

See https://github.com/CareEvolution/MyDataHelpsKit-dev/pull/1#discussion_r598414777 for the original discussion about this.

Reply:
Also, your comment has brought attention to the unnecessary documentation comment "May be one of…", which I copied/pasted from the JS SDK. I'll remove that.